app/components/service/user/user.service.js

```diff
@@ -107,6 +107,10 @@
       setCurrentUser({});
     }
 
+    function authentication() {
+      return getCurrentUser().userType.id == 2;
+    }
```

Comment:
This function returns the administrator permissions to access their specific functions, replace the name getPermissions